settings.go

```diff
@@ -28,6 +30,8 @@ type Settings struct {
 	StatsdPort int `envconfig:"STATSD_PORT" default:"8125"`
 	// Flushing interval.
 	FlushIntervalS int `envconfig:"GOSTATS_FLUSH_INTERVAL_SECONDS" default:"5"`
+	// Whether a LoggingSink should be used is Statsd is not enabled.
+	LoggingSinkEnabled bool `envconfig:"GOSTATS_LOGGING_SINK_ENABLED" default:"false"`
```

Comment:
default to it being enabled, otherwise we break the existing functionality

Reply:
oops sorry, that was my intent, messed this up